main_server/local_server/install_wizard/installer.js

```diff
@@ -556,6 +558,23 @@ function BMLTInstaller( in_prefs    ///< A JSON object with the initial prefs.
         if (this.m_google_api_key_is_good === true && this.m_database_credentials_are_good === true && this.m_server_admin_password_is_good === true) {
             document.getElementById('database_install_stuff_div').className = '';
         }
+    
+        this.m_installer_state.format_lang_names = convertFormatLangsToJSON(document.getElementById('format_lang_names').value);
```

Comment:
The free-form text field for inputting these lends itself to human error. My guess is that this is probably mostly fine since this can be seen as an advanced feature at first, but I think we should do some basic input validation. At the very least, I think this parsing function should be a bit more robust.

1. It could split on whitespace instead of single spaces.
2. It could validate that both parts[0] and parts[1] are not empty strings when trimmed.

It could throw an alert when either of those error conditions are encountered.